### PR TITLE
AppTile improvements

### LINKED ISSUE #1569: AppTile should be exported with withLocales
AppTile uses locales that should not be part of the app
locales, but should be "embedded" in the component.
Otherwise the app has to do that itself.

### PR DESCRIPTION
**Pull request by ptbrowne**
- Exported with withLocales (fix #1569)
- Shown in styleguide
- Tile can be shown in a non flex container

### CODE REVIEW

File: react/AppSections/shared-locales.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/**
+ * This file is present since babel inlines the content of JSON
+ * files where they are imported. Since we need to share
+ * those locales between Sections and AppTile, this file is
```
Comment (Crash--) [inline react/AppSections/shared-locales.js:4]:
```suggestion
 * those locales between AppSections and AppTile, this file is
```

File: react/AppTile/Readme.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+Component used to show an app status, can be used in
```
Comment (Crash--) [inline react/AppTile/Readme.md:1]:
```suggestion
Component used to show an app statu. Can be used in
```

**Review by Crash-- — APPROVED** [review]
Great! 

Wondering if we should not create an `App` folder with AppSections / AppTile and so on since we rely on the same langage files.

**Review by JF-Cozy — APPROVED** [review]

**Comment by ptbrowne** [comment]
> Wondering if we should not create an App folder with AppSections / AppTile and so on since we rely on the same langage files.

I wonder if it would not better instead that they rely on different locale files 🤔 I think it's better if they do not rely on the same file in fact... I am going to split them. This way, no weird shared-locales.js file and through tree shaking, we embark only the locales that we need.

**Pull request merged**